src/PanoImageRenderer/PanoImageRenderer.js

```diff
@@ -65,21 +66,31 @@ export default class PanoImageRenderer extends Component {
 		this.canvas = this._initCanvas(width, height);
 
 		this._image = null;
-		this._imageLoader = new ImageLoader();
 
-		this._onImageLoad = 	this._onImageLoad.bind(this);
-		this._onImageError = 	this._onImageError.bind(this);
```

Comment:
I've replaced `image` with `content`. 

`media` was a candidate.

Reply:
Because **the media element** only contains **video** and **audio**, the name media is not appropriate for image and video.

I'm okay with unified loader name as **content loader**.